source/octf/trace/parser/ParsedIoTraceEventHandler.cpp

```diff
@@ -261,6 +261,16 @@ void ParsedIoTraceEventHandler::handleEvent(
         std::shared_ptr<proto::trace::Event> traceEvent) {
     using namespace proto::trace;
 
+    if (traceEvent->has_filesystemmeta()) {
+        traceEvent->mutable_filesystemmeta()->set_partitionid(271581185);
```

Comment:
What is this number? Please replace this magic number with constexpr or use some defined value if available (proto::type::maxVal maybe?)

Reply:
It should be removed.